install_config/storage_examples/shared_storage.adoc

```diff
@@ -13,4 +13,473 @@ toc::[]
+<7> This is the NFS server. This can also be specified by IP address.
+====
+
+Save the PV definition to a file, for example *_nfs-pv.yaml_*,
+and create the persistent volume:
+
+====
+----
+# oc create -f nfs-pv.yaml 
+persistentvolume "nfs-pv" created
+----
+====
+
+Verify that the persistent volume was created:
+
+====
+----
+# oc get pv
+NAME         LABELS    CAPACITY   ACCESSMODES   STATUS      CLAIM     REASON    AGE
+nfs-pv   <none>    1Gi        RWX           Available                       37s
+----
+====
+
+=== 2. Create the Persistent Volume Claim
+A persistent volume claim (PVC) specifies the desired access mode and storage capacity.
+Currently, based on only these two attributes, a PVC is bound to a single PV. Once a PV is
+bound to a PVC that PV is essentially tied to the PVC's project and cannot be bound to by
+another PVC. There is a one-to-one mapping of PVs and PVCs. However, multiple pods in the same
+project can use the same PVC. This is the use case we are highlighting in this example.
+
+.PVC Object Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs-pvc  <1>
+spec:
+  accessModes:
+  - ReadWriteMany      <2>
+  resources:
+     requests:
+       storage: 1Gi    <3>
+----
+<1> The claim name is referenced by the pod under its `*volumes*` section.
+<2> As mentioned above for PVs, the `*accessModes*` do not enforce access right,
+but rather act as labels to match a PV to a PVC.
+<3> This claim will look for PVs offering *1Gi* or greater capacity.
+====
+
+Save the PVC definition to a file, for example *_nfs-pvc.yaml_*,
+and create the PVC:
+
+====
+----
+# oc create -f nfs-pvc.yaml 
+persistentvolumeclaim "nfs-pvc" created
+
+#and verify the PVC was created and bound to the expected PV:
+# oc get pvc
+NAME            LABELS    STATUS    VOLUME       CAPACITY   ACCESSMODES   AGE
+nfs-pvc         <none>    Bound     nfs-pv       1Gi        RWX           24s
+                                    <1>
+----
+<1> the claim (nfs-pvc) was bound to the "nfs-pv" PV.
+====
+
+=== 3. NFS Volume Access
+Access is necessary to a node in the NFS server. On this node examine
+the nfs-fuse mount:
+
+====
+----
+#on a nfs storage node:
+[root@nfs nfs]# ls -lZ /opt/nfs/
+total 8
+-rw-r--r--. 1 root 100003  system_u:object_r:usr_t:s0     10 Oct 12 23:27 test2b
+              <1>   
+                     <2>
+----
+<1> the owner has id 0.
+<2> the group has id 100003.
+====
+
+In order to access the NFS mount, the container must match the SELinux label, and
+either run with a UID of 0, or with 100003 in its supplemental groups range. It is recommended to gain
+access to the volume by matching the NFS mount's groups, which will be defined in the pod
+definition below.
+
+By default, SELinux does not allow writing from a pod to a remote NFS server. To enable
+writing to NFS volumes with SELinux enforcing on each node, run:
+
+----
+# setsebool -P virt_sandbox_use_nfs on
```

Comment:
In my test env I do not set `virt_sandbox_use_nfs` on.
I only need to set `virt_use_nfs` on. I did some testing related to this and that's the result I got. However, there is discussion in PR #1726 related to these nfs bools. We need to be consistent here.